examples/external-mysql/values.yaml

```diff
@@ -0,0 +1,11 @@
+spire-server:
+  dataStore:
+    sql:
+      database_type: mysql
+      host: mysql
+      port: 3306
+      database: spire
+      username: spire
+      password: CHANGEME
```

Comment:
Maybe add a disclaimer, folks probably want to inject this differently so it doesn't end up in git repo. To also mitigate @faisal-memon his question.

Reply:
oh, great point... It should be removed here and documentation updated.... Will fix.

Reply:
Fix committed.